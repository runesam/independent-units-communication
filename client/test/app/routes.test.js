import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Routes from 'app/routes';

describe('<Routes />', () => {
    it('matches the snapshot', () => {
        const wrapper = shallow(<Routes />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
