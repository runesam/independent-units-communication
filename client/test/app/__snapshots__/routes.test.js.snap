// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches the snapshot 1`] = `
<Provider
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <_default />
</Provider>
`;

exports[`<Routes /> matches the snapshot 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Connect(PublicRoute)
      component={[Function]}
      path="/login"
    />
    <Connect(PrivateRoute)
      component={[Function]}
      exact={true}
      path="/"
    />
    <Connect(PrivateRoute)
      component={[Function]}
      exact={true}
      path="/game"
    />
  </Switch>
</Router>
`;
